apps/dashboard/app/models/posix_file.rb

```diff
@@ -59,6 +42,25 @@ def initialize(path)
     # accepts both String and Pathname
     # avoids converting to Pathname in every function
     @path = Pathname.new(path)
+    @stat = @path.lstat
```

Comment:
current code says: `path.stat will not work for a symlink and will raise an exception` but I cannot replicate.

Reply:
I can replicate now! although it's through calling `lstat` or `stat` (both throw errors) on files that don't exist.

```ruby
irb(main):006:0> Pathname.new('/asdf/stsdfs/sdfdsf').stat
(irb):6:in `stat': No such file or directory @ rb_file_s_stat - /asdf/stsdfs/sdfdsf (Errno::ENOENT)
        from (irb):6:in `stat'
        from (irb):6:in `<main>'
        from /home/jeff/apps/ruby/3.0.6/lib/ruby/gems/3.0.0/gems/irb-1.3.5/exe/irb:11:in `<top (required)>'
        from /home/jeff/apps/ruby/3.0.6/bin/irb:23:in `load'
        from /home/jeff/apps/ruby/3.0.6/bin/irb:23:in `<main>'
```